src/BloomBrowserUI/bookEdit/js/readerSettings.ts

```diff
@@ -26,6 +28,14 @@ class ReaderStage {
   getName(): string {
     return this.name;
   }
+
+  setAllowedWordsString(fileContents: string): void {
+    // split into individual words and sort
+    var words = fileContents.split(/[,;\t\s\r\n]/).sort();
```

Comment:
It can, but in the next line [`_.compact(words)`](http://underscorejs.org/#compact) removes empty items.